File: LeetCode/ProblemSet/Volume (1200-1299)/LeetCode1291/LeetCode1291-Solution-01.cpp
#include <bits/stdc++.h>

class Solution {
public:
    std::vector<int> sequentialDigits(int low, int high) {
        std::vector<int> ans;
        for (int i = 1; i < 10; i++) {
            int res = 0;
            for (int j = i; j < 10; j++) {
                res = res * 10 + j;
                if (low <= res && res <= high) {
                    ans.emplace_back(res);
                }
            }
        }

        std::ranges::sort(ans);

        return ans;
    }
};

File: LeetCode/ProblemSet/Volume (1200-1299)/LeetCode1291/LeetCode1291-Solution-01.py
from typing import *

class Solution:
    def sequentialDigits(self, low: int, high: int) -> List[int]:
        ans = []
        for i in range(1, 10):
            res = 0
            for j in range(i, 10):
                res = res * 10 + j
                if low <= res <= high:
                    ans.append(res)

        ans.sort()

        return ans